src/types/types.ts

```diff
@@ -219,7 +222,7 @@ export type Move = {
   generation: string
   power?: number
   pp: number
-  pastGenPP: {
+  pastGenPP?: {
```

Comment:
If you make this optional, update the lines in `getMoveMaxPP()` in `src/types/pkm/util.ts`